wcfsetup/install/files/lib/system/session/SessionHandler.class.php

```diff
@@ -845,18 +845,11 @@ public function delete() {
 			}
 		}
 		
-		// Delete session.
-		$sql = "DELETE FROM wcf".WCF_N."_".($this->isACP ? 'acp' : 'user')."_session
-			WHERE	sessionID = ?";
-		$statement = WCF::getDB()->prepareStatement($sql);
-		$statement->execute([$this->sessionID]);
-		
-		// Delete legacy session.
-		if (!$this->isACP) {
-			$sql = "DELETE FROM wcf".WCF_N."_session
-				WHERE	sessionID = ?";
-			$statement = WCF::getDB()->prepareStatement($sql);
-			$statement->execute([$this->sessionID]);
+		if ($this->isACP) {
```

Comment:
The `resetSessions` in this method will become obsolete when you adjust the other locations.